lib/modules/manager/custom/regex/readme.md

```diff
@@ -52,6 +52,7 @@ RUN curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version ${YARN_VER
 ```
 
 You would need to capture the `currentValue` with a named capture group, like this: `ENV YARN_VERSION=(?<currentValue>.*?)\\n`.
+To update a version string multiple times in a line: use multiple `matchStrings`, one for each occurrence.
```

Comment:
Is it multiple matchStrings or multiple custom managers?

Reply:
The configuration I am using:

```json5
  customManagers: [
    {
      customType: "regex",
      fileMatch: ["remote-images.txt"],
      matchStrings: [
        // for the version on the left part, ignoring the right
        "# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\s\\S+?:(?<currentValue>\\S+)\\s+\\S+:.+",
        // for the version on the right part, ignoring the left
        "# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\s\\S+?:\\S+\\s+\\S+:(?<currentValue>\\S+)",
      ],
      versioningTemplate: "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}",
    },
  ],
```

For a file that looks like this:

```text
# renovate: datasource=github-tags depName=quay/clair versioning=loose
something:4.7.2    something-else:4.7.2
```

This updates both versions

Reply:
Could you add that as an example?

Reply:
extended